Contributions.md

```diff
@@ -0,0 +1,99 @@
+- [Editor's draft](https://w3c.github.io/pointerevents/)
+- [Github](https://github.com/w3c/pointerevents)
+- participants: [Scott Gonzalez]
+
+## Touch Events
+
+- [Recommendation 10 October 2013](https://www.w3.org/TR/touch-events/)
+- [Editor's draft](https://w3c.github.io/touch-events/)
+- [Github](https://github.com/w3c/touch-events)
+- participants: [Scott Gonzalez]
+
+## Other contributions
+
+- [fixed the behavior](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Oct/0057.html) of `stepUp()` and `stepDown()` for `<input type="number">` based on the behavior of the spinner widget in __jQuery UI__. ([Scott Gonzalez]).
+- [Should a captured pointer send transition events by default?](https://github.com/w3c/pointerevents/issues/61) ([Scott Gonzalez])
+
+# ECMAScript / TC39
+
+## Current proposals:
+
+| Proposal | Champion | Stage |
+|----------|----------|-------|
+| [String padding](https://github.com/tc39/proposal-string-pad-start-end) | [Jordan Harband] | 3 |
+| [Asynchronous Iterators](https://github.com/tc39/proposal-async-iteration) | [Kevin Smith] | 2 |
+| [Observable](https://github.com/zenparsing/es-observable) | [Kevin Smith] & [Jafar Husain] | 1 |
+| [Private Fields](https://github.com/zenparsing/es-private-fields) | [Kevin Smith] | 1 |
+| [Class and Property Decorators](https://github.com/wycats/javascript-decorators/blob/master/README.md) | [Yehuda Katz] & Jonathan Turner | 1 |
+| [`String.prototype.at`](https://github.com/mathiasbynens/String.prototype.at) | [Mathias Bynens] & [Rick Waldron] | 0 |
+| [Function Bind Syntax](https://github.com/zenparsing/es-function-bind) | [Kevin Smith] | 0 |
+
+## Landed features:
+
+| Feature | Version | Champion |
+|---------|---------|----------|
+| `Array.from` & `Array.of` | [ES2015][Array.from] | [Dave Herman] & [Rick Waldron] |
+| `Array#find` & `Array#findIndex` | [ES2015][Array#find] | [Rick Waldron] |
+| `Object.assign` | [ES2015][Object.assign] | [Rick Waldron] |
+| Exponentiantion Operator (`**`) | [ES2016][Exp Operator] | [Rick Waldron] |
+| [`Object.values` & `Object.entries`][Object.entries-proposal] | [ES2016][Object.entries] | [Jordan Harband] |
+
+[ES2015]: http://www.ecma-international.org/ecma-262/6.0/index.html
+[Array.from]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-array.from
+[Array.of]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-array.of
+[Object.entries-proposal]: https://github.com/tc39/proposal-object-values-entries
+[Exp Operator]: https://tc39.github.io/ecma262/2016/#sec-exp-operator
+[Object.entries]: https://tc39.github.io/ecma262/#sec-object.entries
+[Array#find]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-array.prototype.find
+[Object.assign]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-object.assign
+
+## Failed proposals:
+
+| Feature | Champion |
+|---------|----------|
+| [`Math.TAU`](https://esdiscuss.org/topic/math-tau) | [Rick Waldron] |
```

Comment:
the linked discussion is at least interesting and show there's not silver bullet on proposals. Some will become a feature, some will fail. Still, it's good to know we worked on it and good to learn reasons a proposal might not be approved.